core/src/main/java/org/lionweb/lioncore/java/utils/NodeTreeValidator.java

```diff
@@ -1,17 +1,37 @@
 package org.lionweb.lioncore.java.utils;
 
+import java.util.HashMap;
+import java.util.Map;
 import org.lionweb.lioncore.java.model.Node;
 
 public class NodeTreeValidator extends Validator<Node> {
   @Override
   public ValidationResult validate(Node element) {
     ValidationResult validationResult = new ValidationResult();
     validateNodeAndDescendants(element, validationResult);
+    validateNodeAndDescendants(element, validationResult);
```

Comment:
two times the same call?

Reply:
I should get more sleep... sorry about that